Deprecate previous Redis Enterprise integration (#2377)
DataDog/integrations-extras: redisenterprise/README.md

# Redis Enterprise (Deprecated)

> ** This integration is will be deprecated starting on July 19, 2024. Please use the latest [Redis Enterprise Datadog Integration][13] going forward. This new integration exposes all of the latest Redis Enterprise metrics and includes updated dashboards.**

Reviewer: Please move this to the overview section of the README file

Author: Done!

> ** This integration is now deprecated. Please use the latest [Redis Enterprise Datadog Integration](https://app.datadoghq.com/integrations/redis-enterprise) going forward. This new integration exposes all of the latest Redis Enterprise metrics and includes updated dashboards.**

Reviewer: This will need to follow the link formatting similar to your previous pr. Ex: 
[Redis Enterprise Datadog Integration][#] -- this number should be the next number you add for your doc links at the bottom of the page.

At the bottom of the page, add the number [#][link] in the same format as the others

Reviewer: Additionally, this should include that this will be deprecated since this hasn't been deprecated yet. Typically we like to include a date like:

This integration will be deprecated August 2024.. etc.

Author: Thanks, Emily! The latest commit should resolve these requests.

> ** This integration is now deprecated. Please use the latest [Redis Enterprise Datadog Integration](https://app.datadoghq.com/integrations/redis-enterprise) going forward. This new integration exposes all of the latest Redis Enterprise metrics and includes updated dashboards.**

Reviewer: ```suggestion
** This integration is will be deprecated starting x date. Please use the latest [Redis Enterprise Datadog Integration](https://app.datadoghq.com/integrations/redis-enterprise) going forward. This new integration exposes all of the latest Redis Enterprise metrics and includes updated dashboards.**
```

# Redis Enterprise (Deprecated)

![img](https://raw.githubusercontent.com/DataDog/integrations-extras/master/redisenterprise/images/redis-enterprise.jpg)

## Overview

> ** This integration is will be deprecated starting on July 19, 2024. Please use the latest [Redis Enterprise Datadog Integration][13] going forward. This new integration exposes all of the latest Redis Enterprise metrics and includes updated dashboards.**

Reviewer: ```suggestion
 **This integration is will be deprecated starting on July 19, 2024. Please use the latest [Redis Enterprise Datadog Integration][13] going forward. This new integration exposes all of the latest Redis Enterprise metrics and includes updated dashboards.**
```

> ** This integration is will be deprecated starting on July 19, 2024. Please use the latest [Redis Enterprise Datadog Integration][13] going forward. This new integration exposes all of the latest Redis Enterprise metrics and includes updated dashboards.**

Reviewer: ```suggestion
**This integration is will be deprecated starting on July 19, 2024. Please use the latest [Redis Enterprise Datadog Integration][13] going forward. This new integration exposes all of the latest Redis Enterprise metrics and includes updated dashboards.**
```
